src/controllers/articleController.js

```diff
@@ -447,6 +449,53 @@ class ArticleController {
       throw (err);
     }
   }
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} returns an object containing a commented article response
+   */
+  static async commentArticle(req, res) {
+    const commenter = req.userData.id;
+    const slugId = req.params.articleSlug;
+    const { comment } = req.body;
+
+    const authorProfile = await User.findOne({ where: { id: commenter } });
+
+    const findArticle = await Article.findOne({ where: { slug: slugId } });
+
+    if (!findArticle) {
+      return res.status(404).json({
+        message: 'Article not found'
+      });
+    }
+
+    const { id } = findArticle;
+    const { userName, bio, image } = authorProfile;
+
+    const articleComment = await Comment.create({
+      articleId: id,
+      articleSlug: slugId,
+      userId: commenter,
+      comment: comment.trim(),
+    });
+
+    return res.status(201).json({
+      message: 'You have commented on the article',
+      articleComment: {
+        id: articleComment.id,
+        articleSlug: articleComment.articleSlug,
+        comment: articleComment.comment,
+        updatedAt: articleComment.updatedAt,
+        createdAt: articleComment.createdAt,
+        commenter: {
+          username: userName,
+          bio,
```

Comment:
What's the use of bio here?

Reply:
I have referred to the README.
<img width="490" alt="Screen Shot 2019-08-26 at 19 25 15" src="https://user-images.githubusercontent.com/34879554/63709492-4fdab300-c837-11e9-8edd-2a24cf0ad28a.png">

Reply:
But in the response, you did not include `following`

Reply:
I think that one is not necessary!